eng/common/scripts/Create-Package-WorkItem.ps1

```diff
@@ -0,0 +1,47 @@
+
```

Comment:
What is this script doing for us? It looks like it is pretty much just calling the other script

Reply:
I have removed this script now. It was added to try out az devops login, but it's not required anymore.